apps/App.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { enableFreeze } from 'react-native-screens';
+import Example from './Example';
+import * as Test from './src/tests';
+
+enableFreeze(true);
+
+export default function App() {
+  return <Example />;
```

Comment:
Maybe add here something like
```
// return <Test.Test42 />
```
and some comments. So it's possible to use it without sorcery 101.

Reply:
@maciekstosio That's a great idea! https://github.com/software-mansion/react-native-screens/pull/2225/commits/393b1cc263c368569821d8b0ed5149c572c418b8